apply-hensurf-patches.sh

```
@@ -173,6 +173,12 @@ try:
 except Exception as e:
     print(f"⚠️ Could not update search config: {e}")
 PYTHON_SCRIPT
+    if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)